### ISSUE TITLE
Docker: Request Line is too large

### ISSUE THREAD

**Issue by jkreileder** [pgadmin-org/pgadmin4#5390]
I'm using dpage/pgadmin4:6.14. Open "Properties" for a table and then going to the "SQL" tab always fails with: 

```html
<html>
  <head>
    <title>Bad Request</title>
  </head>
  <body>
    <h1><p>Bad Request</p></h1>
    Request Line is too large (4840 &gt; 4094)
  </body>
</html>
```

I.e. the request line is larger than gunicorn's default of 4096.

<img width="1701" alt="image" src="https://user-images.githubusercontent.com/4390544/193415685-a463ce8f-5de9-4ded-aec3-9f29d33e5f65.png">

**Comment by cdhowie** [comment]
@jkreileder I found the same thing. See my report #5392 for a Dockerfile you can use to build a fixed image.

**Comment by dpage** [comment]
Fixed in 992dff64afdffc70125680d14a6fe9d4112ab286
